init.lua

```diff
@@ -5,17 +5,25 @@ if not vim.tbl_contains(vim.opt.rtp:get(), base_dir) then
   vim.opt.rtp:append(base_dir)
 end
 
-require("lvim.bootstrap"):init(base_dir)
+require "lvim.utils.headless_fix"
 
-require("lvim.config"):load()
+-- This is only defined when updating using `lvim --update`
+---@diagnostic disable-next-line: undefined-field
+local updating = _G.__lvim_updating or false
 
```

Comment:
These belong in bootstrap. There shouldn't be any errors from just calling the bootstrap, and if there are, then we have a different problem and the updater shouldn't even run 😅

Reply:
I know you don't like things getting put into init.lua, but see above as to why the `if` guard is necessary.

It is being put into bootstrap.

It just doesn't arbitrarily choose to pick up on a global which may not be set if bootstrap.lua is used from nvim without the init.lua file like you described to me, instead it takes it as a parameter which changes the mode.

Reply:
TLDR: if you want to use update mode elsewhere just use `bootstrap:init(basedir, true)` instead of relying on a global to be set before